core/src/zeit/content/audio/tests/test_reference.py

```diff
@@ -0,0 +1,11 @@
+from zeit.cms.testcontenttype.testcontenttype import ExampleContentType
+import zeit.content.audio.testing
+
+
+class ReferenceTest(zeit.content.audio.testing.FunctionalTestCase):
+    def test_content_audio_references_does_not_break(self):
```

Comment:
```suggestion
    def test_content_audio_references_does_not_break_for_unsupport_content_types(self):
```

Optionale Änderung, wie du magst